README.md

```diff
@@ -55,16 +57,16 @@ Basic commands to check your cluster
 
 1. Download [latest release.zip](https://github.com/microsoft/azure-databricks-operator/releases)
 
+```sh
+VERSION=0.44

```

Comment:
How can we automate this to make sure it always points to the latest version?

Reply:
by using a link similar to: `/owner/name/releases/latest/download/asset-name.zip`

I updated the docs to reference the latest release

[ref](https://help.github.com/en/articles/linking-to-releases)